lib/astarte_flow_web/views/flow_view.ex

```diff
@@ -35,4 +36,22 @@ defmodule Astarte.FlowWeb.FlowView do
   def render("flow_name.json", %{flow: flow}) do
     flow.name
   end
+
+  def render("error.json", %{error: %PipelineBuilder.Error{blocks: blocks}}) do
+    errors =
+      Enum.map(blocks, fn
+        {blockname, {:invalid_block_options, errors}} ->
+          option_errors =
+            Enum.map(errors, fn {message, option} ->
+              %{option: option, message: message}
+            end)
+
+          %{block: blockname, error: "invalid_block_options", option_errors: option_errors}
+
+        {blockname, {error, message}} when is_binary(message) ->
+          %{block: blockname, error: error, message: message}
+      end)
+
+    %{error: "pipeline_error", message: "Failed pipeline instantiation", errors: errors}
```

Comment:
Let's make this consistent with other API errors:
```
%{errors: %{detail: "Failed pipeline instantiation", error: errors}}
```
I'm not a fan of the double `errors`/`error`, the top one should be `errors` for consistency, I'm open to proposals for the inner one (`validation`? `additional_info`?)